domain/blockdag/process.go

```diff
@@ -501,7 +501,7 @@ func (dag *BlockDAG) saveChangesFromBlock(block *util.Block, dbTx *dbaccess.TxCo
 // All blocks in B.MergeSet should be in future of node.finalityPoint.
```

Comment:
`All blocks in B.MergeSet should be in future of node.finalityPoint` This sentence is false. Please rephrase it to be something like `All blocks in B.MergeSet should be in future of node.finalityPoint except...' Or 'All blocks in B.MergeSet should be in future of node.finalityPoint, with the following exception:'